requirements/requirements-ci.txt

```diff
@@ -77,7 +77,6 @@ jinja2==2.10.1
 jsonschema==3.0.1
 lazy-object-proxy==1.4.1
 lru-dict==1.1.6
-macholib==1.14            # via pyinstaller
```

Comment:
I don't understand why this dependency changed, @ulope any ideas?

Reply:
It's a mac only dependency of pyinstaller. pip-tools is unfortunately not good at dealing with plattform specific deps.

Just leave it in.